core/src/test/scala/integration/kafka/api/IntegrationTestHarness.scala

```diff
@@ -80,6 +81,19 @@ abstract class IntegrationTestHarness extends KafkaServerTestHarness {
     }
   }
 
+  private def insertControllerListenersIfNeeded(props: Seq[Properties]): Unit = {
```

Comment:
nit: seems like this would be a bit more natural in `configureListeners` where we already have logic to construct the security map.

Reply:
Unfortunately, this would interfere with the test subclasses that override `configureListeners`.